flink-core/src/main/java/org/apache/flink/util/OptionalMap.java

```diff
@@ -36,7 +36,7 @@
  * An OptionalMap is an order preserving map (like {@link LinkedHashMap}) where keys have a unique string name, but are
  * optionally present, and the values are optional.
  */
-final class OptionalMap<K, V> {
+public final class OptionalMap<K, V> {
```

Comment:
Should we mark it as a `@Internal`?

Reply:
It's already marked as `@Internal` in later commits